package.json

```diff
@@ -40,6 +40,7 @@
     "puppeteer": "19.8.2",
     "stylelint": "15.2.0",
     "stylelint-config-standard": "30.0.1",
+    "stylelint-prettier": "^3.0.0",
```

Comment:
We don't need `stylelint-prettier` plugin (`prettier` docs mentions it at https://prettier.io/docs/en/integrating-with-linters.html#notes)

VS Code workspace has option [editor.formatOnSave](https://github.com/FlowCrypt/flowcrypt-browser/blob/master/flowcrypt-browser.code-workspace#L9), so all `prettier` errors are automatically fixed when saving file, no need for highlighting them.